ingestion_tools/scripts/gjensen_config.py

```diff
@@ -200,12 +200,14 @@ def to_standardization_config(
 
 int_fields = {"ts-tilt_series_quality"}
 float_fields = {
+    "tomo-voxel_spacing",
     "ts-pixel_spacing",
-    "ts-spherical_aberration_constantts-tilt_axis",
+    "ts-spherical_aberration_constant"
+    "ts-tilt_axis",
     "ts-tilt_step",
     "ts-tilt_range-min",
     "ts-tilt_range-max",
-    "ts-total_flux",
+    "ts-total_flux"
```

Comment:
nit: trailing comma is a convention that makes updates easier. :)

Reply:
Fixed!